gravitee-apim-console-webui/src/management/api/api-navigation/api-navigation.component.ts

```diff
@@ -70,6 +71,19 @@ export class ApiNavigationComponent implements OnInit {
   ) {}
 
   ngOnInit() {
+    this.breadcrumbData = {
+      apiName: this.apiName,
+      apiVersion: this.apiVersion,
+      apiState: this.apiState,
+      apiIsSync: this.apiIsSync,
+      apiLifecycleState: this.apiLifecycleState,
+      apiOrigin: this.apiOrigin,
+    };
+    this.subMenuItems.push({
+      displayName: 'Design',
+      targetRoute: 'management.apis.detail.design.policies',
+      baseRoute: 'management.apis.detail.design',
+    });
```

Comment:
You can remove it, it's already done with `this.appendDesign()` (line 89)
_It's probably due to the merge of 3.20.x + a rebase._